rpcs3/Emu/RSX/RSXTexture.h

```diff
@@ -69,6 +69,7 @@ namespace rsx
 		rsx::texture_minify_filter  min_filter() const;
```

Comment:
double space on 4 functions

Reply:
also:
max_aniso

Reply:
looks like this is some old formatting, looking at even more functions above

Reply:
yes, its old formatting